src/test/java/uk/gov/dhsc/htbhf/RunAcceptanceTests.java

```diff
@@ -6,9 +6,9 @@
 
 @RunWith(Cucumber.class)
 @CucumberOptions(plugin = "pretty",
-        features = "src/test/resources/features",
+        features = "src/test/resources/features/acceptance/test-nhs-number.feature",
```

Comment:
You should not be leaving this file like this, this means that only that feature file will be tested, no others, you should only change this for local testing.

Reply:
thanks Mike will do the change.

Reply:
To avoid this in future can you add it to the gitignore file so it won't get checked in